source/guides/tooling/reporters.md

```diff
@@ -19,8 +19,6 @@ Once you've read through the documentation below, we invite you to experience th
 
 # Custom reporter
 
-You need to install any peer dependencies the reporter requires, even if they're bundled with Cypress. For example, {% url "mochawesome" https://github.com/adamgruber/mochawesome %} requires `mocha` as a peer dependency. You will need to install `mocha` as a dev dependency of your own project for it to work.
-
```

Comment:
put in warning block, mention pre-4.4.1

Reply:
We don't mention previous version support in our docs - they should be up to date with current version. 

If you want to note changes, you can add a History table at the bottom to reflect version changes.

Reply:
:+1: